script.js

```diff
@@ -1,4 +1,190 @@
+//====== BEAT THAT GAME ======//
+// Step 1 - 2 Players take turn to play. - Player 1 and 2.
+// Step 2 - When each player clicks submit, rolls 2 dice and show dice number - 2 x Random Nos.
+// Step 3 - Each player picks which 1 of the 2 numbers that they want to specify 1st, and the other as the 2nd number. - 2 configs.
+// Step 4 - After both players have rolled and chosen dice order, player with higher combined number wins. - Comparison of both 2-digit numbers.
+
+// Planning
+// 1) Define game states and global variables.
+// 2) Roll dice function, and convert rolled dice numbers to 2-digit numbers.
+// 3) Create arrays to store numbers.
+// 4) Compare numbers.
+// 5) Reset game.
```

Comment:
Awesome job writing pseudocode and planning what you need to code before starting!